ImageD11/sinograms/point_by_point.py

```diff
@@ -1602,6 +1615,36 @@ def refine_map(refine_points, all_pbpmap_ubis, ri_col, rj_col, sx_grid, sy_grid,
     return final_ubis, final_eps, final_npks, final_nuniq
 
 
+def get_local_gv(i, j, ystep, omega, sinomega, cosomega, xl, yl, zl):
+    """now we correct g-vectors for origin point"""
+    # compute the sample coordinates for this pixel
+    sx, sy = step_to_sample(i, j, ystep)
+
+    # geometry.sample_to_lab_sincos
+    # we only care about the x axis
+    x_offset = sx * cosomega - sy * sinomega
+    # compute local offset
+    xl_local = xl - x_offset
+    # stack xl, yl, zl
+    xyz_local = np.column_stack((xl_local, yl, zl))
+    # hold computed g-vectors
+    gv = np.empty((len(cosomega), 3))
+    # compute g-vectors from xyz
+    ImageD11.cImageD11.compute_gv(xyz_local,
+                                  omega,
+                                  parglobal.get('omegasign'),
```

Comment:
parglobal is not defined inside this function

Reply:
As `parglobal` is globally defined, I don't think it needs to be defined inside this function? It seems to run on my machine, anyway...